pipelines/acs/build.py

```diff
@@ -1,24 +1,34 @@
 from factfinder.main import Pff
-from factfinder.median import get_median, get_median_moe
+from factfinder.multi import Pool
 import os
 import pandas as pd
 import itertools
+from pathlib import Path
+from dotenv import load_dotenv
 
-pff = Pff(
-    api_key=os.environ['API_KEY'], 
-    year = 2018
-)
+# Load .env environmental variables for local runs
+try:
+    env_path = f'{Path(__file__).parent.parent.parent}/.env'
+    load_dotenv(dotenv_path=env_path)
+except:
+    print('.env file is missing ...')
 
+# Initialize pff instance
+pff = Pff(api_key=os.environ['API_KEY'], year = 2018)
 
+# Declare geography and variables involved in this caculation
 geography = ['city', 'tract', 'NTA', 'cd']
-dfs = []
+variables = [(i['pff_variable'], i['domain']) for i in pff.metadata[:10] if i['domain'] != 'decennial']
```

Comment:
Is it possible we'd want to use the domain metadata field to group variables for future projects? That's why I'd thought `['demographic','economic','housing','social']` would be more flexible going forward than `!= 'decennial'`

Reply:
you are right, then we can do `if i['domain'] in ['demographic','economic','housing','social']`